doc/feature/computeVSD.md

```diff
@@ -0,0 +1,202 @@
+number of areas in the area report, then the application will throw an
+exception, as this is a requirement for the correct behaviour of the algorithm.
+
+At this point, all the events will be updated with the VSD value as follows:
+
+> VSD = (V - V0 + G0) * SA * AttenuationFactor
+
+being __V__ the corresponding voltage from the simulation report; __V0__ the
+specified resting potential; __G0__ the specified area multiplier; and __SA__
+the compartment's surface area.
+
+The value of V can be modified, if spike filtering is enabled. For experimental
+purposes, it is possible to set an artificial threshold for the spiking
+activity: if the input voltage goes above the specified threshold value, the
+signal is cropped.
+
+> V = min(V, actionPotentialThreshold)
+
+The attenuation factor is a variable that depends on several factors. If a dye
+curve file is not specified as input, it will always be 1, so no attenuation is
+applied. It also depends on the Y-coordinate of each of the compartment and the
+depth of the loaded circuit (modifiable by an input parameter). Its value is
+computed as follows:
+
+1. The values in the input dye curve file are normalized so they go between 0
+and 1.
+2. If the compartment is above the specified depth value, the first attenuation
+factor in the input dye curve file is applied.
+3. If the compartment is below 0, the last attenuation factor in the input dye
+curve file is applied.
+4. If the compartment is within the specified depth, the circuit space is
+vertically subdivided in as many regions as values are in the file. Each of the
+attenuation factors in the file will be applied to the compartments that fall in
+the corresponding region.
+5. When the interpolate-attenuation option is specified, the attenuation factor
+applied to the VSD value s the result of lineraly interpolating the two closest
+values in the input dye curve file, so the curve is smoother.
+
+
+### Voxelization
+
+Based on two input parameters, _sensor-res_ and _sensor-dim_, we compute the
+desired resolution for the resulting 3D volume, setting the size of the voxels.
+In the X and Z axes, the extent of the volume corresponds to the value specified
+in sensor-dim, in micrometers; and the number of voxels per side is determined
+by sensor-res. The origin of the resulting volume corresponds to the center of
+the bounding box of the soma positions.
+
+All the events are evaluated, computing the corresponding voxel indices based on
+the event center positions, and summing their values together when more than one
+share a voxel index. The original floating point value is kept, being 0 the
+default for empty voxels.
+
+At this point, if the _export-volume_ command line option is specified, the
+resulting 3D volume is written to disk.
+
+
+### Projection of VSD data to the surface of brain
+
+To generate the projection of the data from the 3D volume into a 2D image that
+represents the surface of the brain, the Beer-Lambert law is used. To do that,
```

Comment:
I wouldn't say the 2D image represents the surface of the brain. What you can say is that the 2D plane is on the pial surface of the brain.